corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -45,11 +45,15 @@ def add_arguments(self, parser):
             help='Migrate even if app.vellum_case_management is already true.')
         parser.add_argument('-n', '--dry-run', action='store_true', dest='dry_run',
             help='Do not save updated apps, just print log output.')
+        parser.add_argument('--fail-hard', action='store_true', dest='fail_hard',
+            help='Die when encountering a bad app. Useful when calling via migrate_all_apps_to_cmitfb, '
+                 'which only ever passes a single app id.')
```

Comment:
> migrate_all_apps_to_cmitfb

> which only ever passes a single app id

I'm confused by this comment. Does it migrate all the apps or only one?

Reply:
migrate_all_apps_to_cmitfb migrates all the apps, but it does so by calling migrate_app_to_cmitfb on each individual app